examples/globe_wfs_extruded.html

```diff
@@ -60,10 +89,37 @@
                     globeView.controls.setTilt(45, true);
                 });
             });
-            const d = new debug.Debug(globeView, menuGlobe.gui);
+            var d = new debug.Debug(globeView, menuGlobe.gui);
             debug.createTileDebugUI(menuGlobe.gui, globeView, globeView.wgs84TileLayer, d);
 
-            for (const layer of globeView.getLayers()) {
+            function picking(event) {
+                var htmlInfo = document.getElementById('info');
+                htmlInfo.innerHTML = ' ';
+
+                var intersects = globeView.pickObjectsAt(event, 'WFS Buildings');
+
+                for (var i = 0; i < intersects.length; i++) {
+                    var interAttributes = intersects[i].object.geometry.attributes;
+                    if (interAttributes && intersects[i].object instanceof itowns.THREE.Mesh) {
+                        if (interAttributes.batchid) {
+                            var face = intersects[i].face.a;
+                            var batchid = interAttributes.batchid.array[face];
+                            var batchTable = intersects[i].object.parent.features[batchid].properties;
+                            Object.keys(batchTable).map(function(objectKey) {
```

Comment:
Do you think you could add a more user friendly interface here?
Something that would allow the user to easily get the properties from the picking result.

Reply:
Actually it would make sense to provide a custom `pickObjectsAt` for WFS_Provider.
It would use `Picking.pickObjectsAt()` method, and then would enrich the result by adding an array of properties.

That way, as a user you could do:
```js
for (var i = 0; i < intersects.length; i++) {
    var obj = intersects[i].object;
    var batchTable = intersects[i].properties;
}
```

WDYT?

Reply:
Yes completely, as I said, I would do that in the method `on()`

```js
 function picking(event) {
        const batchTable = event.features[i].properties;
 }
```

Reply:
Yes completely, as I said, I would do that in the method `on()`

```js
featureLayer.on(event) {
        const batchTable = event.features[i].properties;
 }
```

Reply:
Don't you want to do this in this PR (not the `on()` thing, I'm only suggestion to implement my proposal above)?
It would add value to itowns and simplify the example code.